pages/blog/fetch-for-servers.mdx

```diff
@@ -0,0 +1,175 @@
+import withArticle from '../../ui/blog/article'
+
+export const meta = {
+  title: 'JavaScript runs everywhere, so should your servers - here is how',
+  tags: ['fetch', 'whatwg', 'node', 'graphql', 'graphql-yoga'],
+  authors: ['arda'],
+  date: '2022-07-21',
+  description: 'The new way of building servers in JavaScript',
+  image:
+    'https://user-images.githubusercontent.com/727224/104328926-56b0a700-54ba-11eb-9ad3-f6d2b971d55d.jpg',
+  thumbnail:
+    'https://user-images.githubusercontent.com/727224/104328777-29fc8f80-54ba-11eb-87d0-9982af1d8281.jpg'
+}
+
+export default withArticle(meta)
+
+At the beginning of the year, we’ve launched GraphQL Yoga 2.0 - a server framework for GraphQL APIs.
```

Comment:
link to the blog post about graphql yoga

Reply:
```suggestion
At the beginning of the year, [we launched GraphQL Yoga 2.0](https://www.the-guild.dev/blog/announcing-graphql-yoga-v2) - a server framework for GraphQL APIs.
```